src/requests/getForecast.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable no-console */
+import axios from "axios";
+
+const getForecast = (
+  setSelectedDate,
+  setForecasts,
+  setLocation,
+  searchText,
+  setErrorMessage
+) => {
+  let endpoint = "https://cmd-shift-weather-app.onrender.com/forecast";
+
+  if (searchText) {
+    endpoint += `?city=${searchText}`;
+  }
+
+  return axios
+    .get(endpoint)
+    .then((response) => {
+      setSelectedDate(response.data.forecasts[0].date);
+      setForecasts(response.data.forecasts);
+      setLocation(response.data.location);
+      setErrorMessage("");
+    })
+    .catch((error) => {
+      const { status } = error.response;
+      if (status === 404) {
+        setErrorMessage("No such town or city, try again!");
+        console.error("Location is not valid", error);
+      }
+      if (status === 500) {
+        setErrorMessage("Oops, server error, try again later.");
+        console.error("Server error", error);
```

Comment:
Since you are already displaying an error message to the user explaining why things are not working, these console logs could be removed
You could also remove the eslint no console exception at the top of the file as a result of this too